### PR TITLE
[Chore] footer links

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

**Description**

This PR is incomplete, but advances @debo19's work. @debo19 can carry this PR forward to completion. 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit f8fd27791a6fa411241977224411c51b28b75af2 at: https://62ba9dc797af462a9c0df09a--layer5.netlify.app

File: src/sections/General/Footer/index.js [hunk 1]
```diff
@@ -194,6 +210,7 @@ const Footer = () => {
                       Partners
                     </Link>
                   </li>
+                  <div className="divider"> - - - </div>
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply src/sections/General/Footer/index.js:213]: What?! You dare question me? Hoity Toity Supreme? 😤 

My thinking was that the individual columns are becoming quite lengthy and that a divider might help highlight one item out of each segment. Having said this, I was just experimenting, and so please proceed as you see fit.

**Comment by l5io** [comment]
🚀 Preview for commit 7ae4804b1816cdf75fc9badddeb80a098a44d81f at: https://62bb671ca1f29507d2515ebb--layer5.netlify.app

File: src/sections/General/Footer/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (debo19) [reply src/sections/General/Footer/index.js]: 😂 Oh okay. Other than MeshMap, the other columns don't really have an item to highlight(in my opinion). The label does the job for MeshMap, so we can avoid the divider for now.

**Comment by debo19** [comment]
Merging...

**Review by debo19 — APPROVED** [review]

**Pull request merged**